tools/add_triage_signature.py

```diff
@@ -301,7 +301,9 @@ def _update_ticket(self, url, issue_key, should_update=False):
                 progress=host['progress']['current_stage'],
                 status=host['status'],
                 role=role,
-                status_info=str(info)))
+                status_info=str(info),
+                logs_info=host['logs_info'],
+                last_checked_in_at=format_time(host['checked_in_at'])))
```

Comment:
Those fields are always available? (meaning have defaults)
Because otherwise you should do e.g. ``host.get('checked_in_at')``

Reply:
Next PR I guess

Reply:
https://github.com/openshift-assisted/assisted-installer-deployment/pull/193